kbase-extension/static/kbase/js/widgets/appWidgets2/view/structView.js

```diff
@@ -211,79 +191,58 @@ define([
                         return structFields[fieldName].instance.stop();
                     })
                 ).then(() => {
-                    ui.setContent('input-container.subcontrols', '');
+                    ui.setContent('input-container', '');
                     structFields = {};
                 });
             }
         }
 
-        function renderStruct() {
-            const layout = div(
-                {
-                    style: {
-                        'border-left': '5px solid silver',
-                        padding: '2px',
-                        margin: '6px',
-                    },
-                },
-                [div({ dataElement: 'subcontrols' })]
-            );
-            ui.setContent('input-container', layout);
-        }
-
-        function render(events) {
+        function render() {
             container.innerHTML = div(
                 {
                     dataElement: 'main-panel',
                 },
-                [div({ dataElement: 'input-container' })]
+                div({
```

Comment:
Restructured from 3 nested, unused divs to just 1. Really, it could probably all go under the `main-panel` element, but I think that's similar to how the other inputs act, so I left it.